documentation20/en/08.connector/docs.md

```diff
@@ -610,11 +610,11 @@ The return value is in JSON format, as follows:
 ```json
 {
     "status": "succ",
-    "head": ["ts","current", …],
```

Comment:
I briefly looked through the docs, finding these characters weird, so I changed them with half-width `.`